frontend/css/brainzplayer.less

```
@@ -412,3 +412,144 @@
     }
   }
 }
+
+.music-player {
+  position: fixed;
+  padding: 0.5rem;
+  z-index: 9997;
+  height: 100vh;
+  width: 100%;
+  display: flex !important;
+  flex-direction: column;
+  right: 0;
+  bottom: -100%;
+  transition: bottom 1s ease-in-out;
+
+  &.show {
+    bottom: 0;
+  }
```

I saw that the .show class has a `display:... !important` which comes from Bootstrap, forcing you to use `!important` as well here.
But you can use a different class name and avoid that easily:

```suggestion
  display: flex;
  flex-direction: column;
  right: 0;
  bottom: -100%;
  transition: bottom 1s ease-in-out;

  &.open {
    bottom: 0;
  }
```

---

fixed